akka-stream/src/main/scala/akka/stream/impl/ActorProcessor.scala

```diff
@@ -250,7 +250,8 @@ private[akka] abstract class ActorProcessorImpl(val settings: MaterializerSettin
   protected def onError(e: Throwable): Unit = fail(e)
 
   protected def fail(e: Throwable): Unit = {
-    log.error(e, "failure during processing") // FIXME: escalate to supervisor instead
+    // FIXME: escalate to supervisor
```

Comment:
I am not sure about the escalate part. Also we have a ticket about supervision, this FIXME is therefore superceded by that.